machine/dni/dni_3048up/kernel/platform-dni_3048up.patch

```diff
@@ -0,0 +1,1933 @@
+From 8cdecead792e797e8ecaf8f50f18cbec08d56a9e Mon Sep 17 00:00:00 2001
+Subject: [PATCH] Support for Agema AG6248C-POE platform (ARM Cortex-A9 /
+ Helix4) networking platform Baseline files sourced from
+ Broadcom LDK 3.4.10
+
+diff --git a/arch/arm/boot/dts/agema_ag6248c-poe.dts b/arch/arm/boot/dts/agema_ag6248c-poe.dts
+new file mode 100644
+index 0000000..d229a6e
+--- /dev/null
++++ b/arch/arm/boot/dts/agema_ag6248c-poe.dts
+@@ -0,0 +1,73 @@
++/*
++ * Agema AG6248C-POE Device Tree Source
++ *
++ * Copyright 2015, Agema, Inc.
++ *
++ * This program is free software; you can redistribute  it and/or modify it
++ * under  the terms of  the GNU General  Public License as published by the
++ * Free Software Foundation;  either version 2 of the  License, or (at your
++ * option) any later version.
++ *
++ */
++/dts-v1/;
++/include/ "helix4.dtsi"
++
++/ {
++    model = "agema,ag6248c-poe";
++    compatible = "agema,ag6248c-poe";
```

Comment:
These names should match the ONIE machine name.  It should say:  "dni,dni_3048up".

See the design spec for more on the use of these fields:
https://github.com/opencomputeproject/onie/wiki/Design-Spec-U-Boot-Linux-Kernel